src/plugins/console/public/lib/utils/index.ts

```diff
@@ -175,3 +176,9 @@ export const replaceVariables = (
     return req;
   });
 };
+
+const isUuid = (value: string) => {
```

Comment:
Don't we use uuidv4 got generating these? If so we could just use the method provided already by the library 
```
import { isUuid } from 'uuidv4';
```

Reply:
Looks like we do, so we could use the method provided by the library, however, it looks like we don't have this library as a dependency. We have `uuid` which does have a `validate` method, but the version we have doesn't expose it. We could update the version, but I'm not sure if that's a good idea.

Reply:
Yeah I think it's a good idea! Nice find!